docs/design/per-op-config.md

```diff
@@ -167,13 +221,14 @@ suspend fun main() {
 }
 ```
 
-The final accepted design ultimately evolved from this. We argue that there's no reason to restrict the user to making
-calls within this forced scope - we can allow the caller to save and pass around the returned client as long as we're
-responsible with SDK-managed closeable resources that are shared throughout.
+This alternative was rejected in favor of the proposed design since it accomplishes the same thing while expanding the
+scope of use. the SDK can allow the caller to save and pass around the returned client as long as the runtime is
+responsible with managed closeable resources that are shared throughout. The use of two lambda arguments was also deemed
+non-idiomatic.
 
 Additionally, the pattern enforced by this API (client only lives within `useBlock`) can still be expressed in the
 accepted design simply by making the `use { }` call on the client yourself, as demonstrated in its example usage.
```

Comment:
**Nit**: Avoid "yourself". Consider "itself" (changes the meaning of the phrase slightly but accurately).